server/vulnerabilities/sync.go

```diff
@@ -0,0 +1,262 @@
+package vulnerabilities
+
+import (
+	"context"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/facebookincubator/nvdtools/cvefeed"
+	feednvd "github.com/facebookincubator/nvdtools/cvefeed/nvd"
+	"github.com/fleetdm/fleet/v4/pkg/download"
+	"github.com/fleetdm/fleet/v4/pkg/fleethttp"
+	"github.com/fleetdm/fleet/v4/server/config"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/fleetdm/fleet/v4/server/ptr"
+	"github.com/hashicorp/go-multierror"
```

Comment:
`multierror` and `multierr`?

Reply:
Is this ok then  https://github.com/fleetdm/fleet/pull/5838/files/30dcd2fb8bb92eb957d3598cff04778f90da3f78#diff-b4fccdf3817b1ee9ea965b77019b9e91120e397d7d255875227ef75cbccb9f1eR38 ?

Reply:
the change is marked as outdated in the github conversation. Latest version does not use it.